azurelinuxagent/common/protocol/wire.py

```diff
@@ -232,6 +244,7 @@ def _build_health_report(incarnation, container_id, role_instance_id,
            u"<GoalStateIncarnation>{0}</GoalStateIncarnation>"
            u"<Container>"
            u"<ContainerId>{1}</ContainerId>"
+           u"<Users>{5}</Users>"
```

Comment:
Why put a {5} between a {1} and a {2}?

Reply:
Only because I'm less likely to fat finger it this way vs shifting every number (string interpolation makes this more logical, but it doesn't seem Python has that.  I've changed this now.